roleGroup.py

```diff
@@ -96,3 +96,12 @@ def fromJson(self, data: dict, emojis: Tuple[Emoji], manual_load=False):
         if manual_load:
             # Remove roles that were not present in imported data
             self.roles = [x for x in self.roles if x.emoji in roles]
+
+#KOSI START this only needs to be loaded once
+    def getCorrectedName(self, roleData):
+        if "displayName" in roleData:
+            show_name =  roleData["displayName"]
+        else:
+            how_name = roleData["show_name"]
```

Comment:
```suggestion
            show_name = roleData["show_name"]
```

Reply:
yea I saw that later but forgot to push that commit